prerendering.bs

```diff
@@ -658,6 +673,41 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+We need to ensure that the [=Accept-CH cache=], which is owned by user agent as a global storage, is not modified while prerendering, but is properly updated after activation. The following modifications ensure this:
```

Comment:
nit: I'd probably just use a period (`.`) instead of a colon (`:`) here, since it's really introducing the entire section and not a bulleted list.

Reply:
Thanks! Done.